src/ftp/tests/listener_endpoint_test.bal

```diff
@@ -0,0 +1,44 @@
+import ballerina/log;
+
+listener Listener remoteServer = new({
+    protocol: FTP,
+    host: "192.168.112.16",
```

Comment:
Is this FTP endpoint available all the time? Why shouldn't we make this configurable?

Reply:
This is fixed in #55.